.gitignore

```diff
@@ -1,5 +1,6 @@
 TODO
 coverage
+dist
```

Comment:
yeah, some repos dont include dist but see why yours does. I can live with the git status changes when it builds